src/views/Users/Login/Login.vue

```diff
@@ -24,6 +24,11 @@
             <v-card-text>
               <!-- message handler -->
               <MessageHandler field="login" />
+
+              <!-- button to re-send confirmation if login failed -->
+              <span v-show="resendButton">
```

Comment:
use v-if instead of v-show.
the difference if that v-if rebuild that part of the DOM where v-show is just a css display: none attribute